pkg/entrypoint/termination

```diff
@@ -0,0 +1 @@
+[{"key":"StartedAt","value":"2021-08-19T11:40:06.469+10:00","type":"InternalTektonResult"},{"key":"ExitCode","value":"2","type":"InternalTektonResult"},{"key":"StartedAt","value":"2021-08-19T11:40:06.504+10:00","type":"InternalTektonResult"},{"key":"StartedAt","value":"2021-08-19T11:40:06.519+10:00","type":"InternalTektonResult"},{"key":"StartedAt","value":"2021-08-19T11:40:06.531+10:00","type":"InternalTektonResult"}]
```

Comment:
This is an artifact of a unit test in `pkg/entrypoint` and should be removed I think.  If you rebase on `main` and re-run `go test ./...` it shouldn't appear again.

Reply:
Good catch :) I'm pretty sure I've rebased on main recently, so I'll remove that file...